packages/frontend-shared/src/components/Card.vue

```diff
@@ -17,6 +17,13 @@
     >
       {{ t('versions.beta') }}
     </div>
+    <div
+      v-if="title === t('createSpec.component.importFromComponent.header')"
```

Comment:
This seems strange - there must be a conditional somewhere up the chain that decides `title`. Instead of doing a conditional on a string, I wonder if we should find out what the actual logic is and either pass a prop or use it here?

Reply:
I moved this logic up out of `Card` and made it a prop that the parent component passes down. If there's badge text, then we show the badge, otherwise we don't.